ff-outro.md

```diff
@@ -0,0 +1,89 @@
+
+This asynchronous workforce model defies the conventional way practically unchanged from the First Industrial Revolution. It better embraces the Information Age, especially with the catalyst known as the internet. The model leads to realizations of how elaborate choices of where and when to work not only do not diminish productivity, but by respecting each others' time, life, and work style, we see less burnout and more innovations spark out.
+
+The synchronous forms of communication do not entirely vanish. They become more scarce and thus more treasured by each other. When participants resolve most motions asynchronously, the calls and meetings become laser-focused on the unresolved remainders, preserving time and effort. Equally importantly, the in-person times serve a great socialization purpose to remind us that we are still humans.
+
+### Automation and Hacking
+
+To sustain an asynchronous and decentralized workforce with a massive amount of information exposed through transparency, open-source participants frequently make and improve automation to optimize or abstract away processes. This mindset drives people to continually lookout for new ways to adapt and improve their workflow.
+
+But it is rare for existing tools and automation to provide a perfect fit for everyone's use-cases. People resort to hacking, or creative workarounds, to bend tools to their will.
+
+The hacking culture is common in making open-source software itself, providing an organic counter-balance to some problems that plague many projects, such as under-documentation, lack of testing, and [leaky abstractions](https://en.wikipedia.org/wiki/Leaky_abstraction).
+
+## git
+
+Among many tools that support the open-source software ecosystem, `git` is the most popular one that provides efficient ways to distribute and manage source code. Its decentralized nature allows every contributor to fully access the working copy and the entire change history and derivatives.
+
+The collaboration happens in a peer-to-peer fashion. Collaborators branch out the source code to implement new features or issue fixes. Upon concluding the implementation, the contributor requests to merge back to the mainstream working copy of the source code, attract others to review, and offer suggestions to improve. When enough contributors reach a consensus, the implementation gets merged back to the mainstream. During this entire process, automation plays a critical role in simplifying every participant's experience through automated testing and suggestions, so contributors do not waste time on trivial problems such as code-style differences and logical flaws. After the merge, it is common to see automated deployment or release of the software to a designated audience.
```

Comment:
is `branch out` a jargon that we should further clarify for non-devs? Maybe `fork` and `merge` could be illustrated with those [diagrams](https://www.bogotobogo.com/cplusplus/images/Git/Fast_Forward_Merge/TypicalMerge.png ) images - from https://www.bogotobogo.com/DevOps/SCM/Git/Git_GitHub_Fast-Forward_Merge.php?

Reply:
You’re right, image illustrations would help a lot! I’ll add those later